src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+const populationElements = document.querySelectorAll('.population');
```

Comment:
It is a good practice to avoid using 'magic strings' in your code. 'Magic string' is a term for string that has some underlying meaning in our code. In your case '.population', '.total-population', and '.average-population' are magic strings. Consider declaring them as constants at the top of your file, this way it is easier to manage and less prone to typos.